script.js

```diff
@@ -0,0 +1,13 @@
+const mobileMenu = document.getElementById('mobile-menu');
+const closeBtn = document.getElementById('closeMenu');
+const menu = document.getElementById('menu');
+
+menu.onclick = function showMenu() {
+  mobileMenu.style.display = 'block';
+};
+closeBtn.onclick = function closeMenu() {
+  mobileMenu.style.display = 'none';
+};
+mobileMenu.onclick = function close() {
+  mobileMenu.style.display = 'none';
+};
```

Comment:
- [ ] [REQUIRED] Good job so far 👍  I appreciate what you have done, but I found that there is a problem with the `mobile menu`, so when you change your project to the `mobile version`, the mobile menu is `automatically` launched, please fix this, because the `mobile menu` should only be `launched` when the `hamburger` button is clicked 🙏 

![Bug-mob-menu](https://user-images.githubusercontent.com/63915024/202010355-5663a370-9ab4-4b40-b72c-c673cc0209e6.gif)